source/tyr/tyr/resources.py

```diff
@@ -1113,3 +1116,28 @@ def delete(self, name=None):
             raise
         return ({}, 204)
 
+class Data(flask_restful.Resource):
+    def post(self, instance_name):
```

Comment:
Maybe this function should be part of the `Job`api since the purpose of it is to create a new job.